horace_core/sqw/file_io/@faccess_sqw_v4/get_sqw.m

```diff
@@ -68,9 +68,18 @@
 [obj,sqw_skel] = obj.get_all_blocks(sqw_skel,'ignore_blocks',skip_blocks);
 
 if ~(opts.head || opts.his)
+    detpar = sqw_skel.detpar;
+    if ~isempty(detpar)        
+        detpar = IX_detector_array(detpar);
+    else
+        detpar = IX_detector_array();
+    end
```

Comment:
No, line 75 is not equivalent to IX_detector_array([]) which would give nargin==1. It would be possible to conflate these with a refactor of IX_detector_array and IX_detector_bank, but not now